### PR TITLE
Adding DELETE/testimonials/:id

### PR DESCRIPTION
**Pull request by Lucasfranco1**
**SUMARY**

- Added method to delete a testimonial
- Added exceptions package
- Added RestExceptionHandler class to unify exceptions.
- Added ParamNotFound class in exceptions package
- Modification of the validate method to use own exceptions.
- Added path in SecurityConfig for user admin

### CODE REVIEW

**Comment by JonathanBolatti** [comment]
thanks for adding the class to implement errors. the code looks great, do you need to add anything else?

**Comment by Lucasfranco1** [comment]
> Thanks for adding the class to implement errors. the code looks great, do you need to add anything else?
Thanks bro! No, I'm waiting for the merge to be done so I can finish the other ticket.

File: src/main/java/com/alkemy/ong/controller/RestExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+@ControllerAdvice
+public class RestExceptionHandler extends ResponseEntityExceptionHandler {
```
[the remark this answers is not in the document]
Comment (Lucasfranco1) [reply src/main/java/com/alkemy/ong/controller/RestExceptionHandler.java:21]: Thanks!

**Comment by JonathanBolatti** [comment]
approved

**Pull request merged**